<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::CertificateDialog</class>
 <widget class="QDialog" name="Otter::CertificateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,2,0,3,0,1">
   <item>
    <widget class="QLabel" name="chainLabel">
     <property name="text">
      <string>Certificate chain:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Otter::ItemViewWidget" name="chainItemView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="fieldsLabel">
     <property name="text">
      <string>Certificate fields:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Otter::ItemViewWidget" name="detailsItemView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="valueLabel">
     <property name="text">
      <string>Field value:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="valueTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::ItemViewWidget</class>
   <extends>QTreeView</extends>
   <header>src/ui/ItemViewWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
